client/src/components/GoogleMaps.vue

```diff
@@ -151,16 +125,17 @@ export default Vue.extend({
       const { RIGHT_BOTTOM } = google.maps.ControlPosition;
       this.map.controls[RIGHT_BOTTOM].push(centerControlDiv);
     },
-    selectQRspot({ lat, lng }, index) {
-      this.map.panTo(new google.maps.LatLng(lat, lng));
-      this.markerSelected = index;
-      this.currentSpot = this.currentMarker;
-      this.showSpotInfo = true;
+    selectQRspot(marker) {
+      this.map.panTo(new google.maps.LatLng(marker.lat, marker.lng));
+      this.$store.commit("qrSpot/setQrSpot", marker);
+      this.$store.commit(
+        "qrSpot/setShowQrSpot",
+        this.modalStateQRspot.SHOW_INFO
+      );
```

Comment:
Use `mapMutations`?

Reply:
Can you have another look now and see if is correct?

Reply:
:heart: